src/fu-device.c

```diff
@@ -1725,6 +1725,7 @@ fu_device_incorporate (FuDevice *self, FuDevice *donor)
 	FuDevicePrivate *priv = GET_PRIVATE (self);
 	FuDevicePrivate *priv_donor = GET_PRIVATE (donor);
 	g_autoptr(GList) metadata_keys = NULL;
+	GPtrArray *parent_guids = fu_device_get_parent_guids (donor);
```

Comment:
Normally the g_autoptr's are last in the variable list.

Reply:
Fixed